bcipy/helpers/copy_phrase_wrapper.py

```diff
@@ -57,15 +60,26 @@ def __init__(self, min_num_seq, max_num_seq, signal_model=None, fs=300, k=2,
         seq_constants = []
         if backspace_always_shown and BACKSPACE_CHAR in alp:
             seq_constants.append(BACKSPACE_CHAR)
+
+        # Stimuli Selection Module
+        stopping_criteria = CriteriaEvaluator(
+            continue_criteria=[MinIterationsCriteria(min_num_seq)],
+            commit_criteria=[MaxIterationsCriteria(max_num_seq),
+                             ProbThresholdCriteria(decision_threshold)])
```

Comment:
If we want to use different stopping criteria or use a different Agent, how would we do that? It seems like we would need to repeat a lot of code that's in this constructor. It may be worth breaking this init method up into smaller methods that then get called from here.

Reply:
Since stopping condition right now is a module, just like the stimuli agent, it should be passed to decision maker from a parameter file. I am not sure how to do that and need your assistance. The structure of the stopping criteria was already designed including a continue criteria list and a commit criteria list. I think this also refactoring.

Reply:
I will also need assistance on breaking the the init method into smaller chunks.

Reply:
I'm okay leaving this one and addressing it in another ticket. How can we further simplify constructing these criteria is the ask. It will take some time to think about.